requirements/requirements-core.txt

```diff
@@ -8,3 +8,4 @@ torch # Remove when subpackage functionality is fixed.
 pyssht
 healpy
 ducc0
+nanobind >=2.0.0
```

Comment:
We should be able to remove this file now anyway, but I don't think we have a runtime dependency on `nanobind` anyway?

Reply:
No it is only build time
I will remove it